system/handlers/admin/datamanager/formbuilder_question.cfc

```diff
@@ -188,6 +199,45 @@ component extends="preside.system.base.AdminHandler" {
 		}
 	}
 
+	private string function preRenderRecord( event, rc, prc, args={} ) {
+		var recordId = prc.recordId ?: ( rc.recordId ?: "" );
+		var isInUse  = formBuilderQuestionService.questionIsInUse( questionId=recordId );
+
+		if ( isTrue( isInUse ) ) {
+			return renderView( view="/admin/formbuilder/_questionInUseWarning", args=args );
+		}
+
+		return "";
+	}
+
+	private void function preDeleteRecordAction( event, rc, prc, args={} ) {
+		var records = args.records ?: QueryNew('');
+
+		for ( var record in records ) {
+			if ( formBuilderQuestionService.questionIsInUse( questionId=record.id ) ) {
+				messageBox.error( translateResource( uri="preside-objects.formbuilder_question:multiAction.question.in.use.warning" ) );
+				setNextEvent( url=event.buildAdminLink( objectName="formbuilder_question" ) );
+			}
+		}
+	}
+
+	private void function extraRecordActionsForGridListing( event, rc, prc, args={} ) {
+		var objectName = args.objectName ?: "";
+		var record     = args.record     ?: {};
+		var recordId   = record.id       ?: "";
+		var isInUse    = formBuilderQuestionService.questionIsInUse( questionId=recordId );
+
+		if ( isTrue( isInUse ) ) {
+			args.actions = args.actions ?: [];
+
+			for ( var action in args.actions ) {
+				if ( find( "deleteRecordAction", action.link ?: "" ) ) {
+					arrayDelete( args.actions, action );
```

Comment:
Is there a way we can *disable* this instead of deleting it? Removing it messes with alignment in the grid. Perhaps: 

`.link = "##", class="disabled" ` (or similar)